### PR TITLE
add CAPG office hour

### PR DESCRIPTION
**Pull request by cpanato**
adding the CAPG office hours

I need to know how to create a zoom meeting and maybe have the host key?

/assign @dims @neolit123 @mrbobbytables

### CODE REVIEW

**Comment by mrbobbytables** [comment]
@cpanato it should be created by the sig leads for SCL and they can provide you with info / host key.

File: sig-cluster-lifecycle/README.md [hunk 1]
```diff
@@ -102,6 +102,10 @@ The following [subprojects][subproject-definition] are owned by sig-cluster-life
 ### cluster-api-provider-gcp
 - **Owners:**
   - [kubernetes-sigs/cluster-api-provider-gcp](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/master/OWNERS)
+- **Meetings:**
+  - Cluster API Provider GCP office hours: [Wednesdays at 10:00 PT (Pacific Time)](TBD) (monthly on 1st Wednesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
```
Comment (neolit123) [inline sig-cluster-lifecycle/README.md:106]: If you are sure about this cadence just remove the TBD.

Comment (cpanato) [reply sig-cluster-lifecycle/README.md]: the `TBD` i guess is the zoom URL

Comment (neolit123) [reply sig-cluster-lifecycle/README.md]: we are locked outside of our Zoom account currently:
https://kubernetes.slack.com/archives/C1TU9EB9S/p1649180197245759

Comment (cpanato) [reply sig-cluster-lifecycle/README.md]: thanks!

Comment (neolit123) [reply sig-cluster-lifecycle/README.md]: we got our zoom acc access back, thanks to @mrbobbytables
let me create a meeting for CAPG and message you on slack about details.

**Comment by neolit123** [comment]
@cpanato we seem to have a problem with the SIG calendar and the event cannot be added there. but at least you can add the Zoom link here: https://zoom.us/j/96963829102?pwd=WjBZcmwvZFZsUU93aVZieUk1L3FnZz09

**Comment by cpanato** [comment]
@neolit123 rebased PTAL

also i did not see the meeting show up in my calendar, there is anything to do?

File: sigs.yaml [hunk 2]
```diff
@@ -1080,6 +1080,15 @@ sigs:
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-gcp/master/OWNERS
+    meetings:
+    - description: Cluster API Provider GCP office hours
+      day: Thursday
+      time: "10:00"
+      tz: PT (Pacific Time)
+      frequency: monthly on 1st Thursday
```
Comment (neolit123) [inline sigs.yaml:1088]: Hm, you had it on Wed earlier. See here:
https://github.com/kubernetes/community/pull/6582#pullrequestreview-929071725
Its scheduled for today.

What day is preferable?

**Comment by neolit123** [comment]
> also i did not see the meeting show up in my calendar, there is anything to do?

There is a meeting in the SIG CL calendar for today (same time as CAPI main meeting), but it did not appear in the k8s community calendar. #sig-contribex  might be able to help.

When adding the meeting I was seeing some odd errors like "quota exceeded" but don't remember exactly. That might be related.

File: sigs.yaml (hunk 2 above)
Comment (cpanato) [reply sigs.yaml:1088]: yes, was my mistake, it should be in the first Thursday of the month

**Comment by mrbobbytables** [comment]
> When adding the meeting I was seeing some odd errors like "quota exceeded" but don't remember exactly. That might be related.

That means too many calendar events have been made by a single individual in too short a time. The only options are to wait or have someone else create the event. This is the same issue that made us move off public google groups to a managed domain for kdev.

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

@cpanato 
i updated the event to be on the first Thursday of the month (starting 5th of may)
but i cannot include the SIG CL ML so that this is synced with the k8s comm calendar.

you can poke `@fabriziopandini` or `@vincepri` on Slack to:
- find the CAPG event on may 5 in the SIG CL calendar
- edit it
- add kubernetes-sig-cluster-lifecycle@googlegroups.com as guest 
- save it and see if it works without error

**Comment by cpanato** [comment]
@mrbobbytables have the host key and just missing the calendar entry which we are trying to get it, but i guess we can merge this PR

**Comment by fabriziopandini** [comment]
@mrbobbytables @cpanato @neolit123 
It seems that there is a problem with the SIG SCL calendar, the same @neolit123 quoted above: 
- we cannot include the SIG CL ML as a guest, because when doing this we got an error about reaching calendar limits.
- as a consequence, the event does not show up in the k8s calendar

As a temporary solution, people can follow the SCL calendar (https://calendar.google.com/calendar/u/0?cid=dTU4MjJscmw0cTY4aWMxaWFrdXZjcGU3YjRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ), but in the long run 2e need guidance on how restore the operativity on the SCL calendar 

However, this PR can merge IMO

**Comment by mrbobbytables** [comment]
@fabriziopandini - the only complete solution to that problem is to migrate off a public googlegroup. =/ This is why we did the kdev migration to a kubernetes.io managed domain. We could no longer send it calendar invites. 
Migrating users is a bit of a pain right now. Theres a hacky script that was used for kdev that is by no means ready for general use, we could bump the priority on getting that taken care of, but it'd definitely be post kubecon.

**Pull request closed without merging**